recipe/meta.yaml

```diff
@@ -137,3 +132,5 @@ extra:
     - xhochy
     - cjmartian
     - dbast
+    - jezdez
+    - chenglee
```

Comment:
Shouldn't this be @chenghlee? (Note the missing 'h')

Reply:
🤦 Thanks both :)